### ISSUE TITLE
Cloud environment with multiple application instances

### ISSUE THREAD

**Issue by wesoos** [brettwooldridge/HikariCP#1679]
My application is deployed in Azure via the App Service with multiple instances.  Should each instance have a unique Hikari pool name?

**Comment by quaff** [comment]
I don't think so, unique name should be used if you have multiple pools in every instance.
